app/hallebarde/responses/authorizer_responses.py

```diff
@@ -0,0 +1,16 @@
+def generate_policy(is_valid: bool, resource: str) -> dict:
+    return {
```

Comment:
Also, did you look at this https://boto3.amazonaws.com/v1/documentation/api/latest/guide/iam-example-policies.html ? For AWS policy generation with boto3

Don't know if it is what we are doing here or totally something else

Reply:
Unfortunately it is not. This is the API for CRUD on IAM policies, but from what I understand it does not provide any utilities for writing such policies (which can be written as dicts in python)